### ISSUE TITLE
`werf run` alternative for running containers in K8s-cluster

### ISSUE THREAD

**Issue by ilya-lesikov** [werf/werf#4280]
Implement alternative for `werf run`, which will run Pod in K8s cluster instead of running local Docker container.

**Comment by ilya-lesikov** [comment]
### TL;DR: new command `werf kube-run` — just like `werf run`, but instead of running local container it runs Pod in K8s-cluster.

### New command: `werf kube-run`

Since v1.2.79 new command `werf kube-run`, just as existing `werf run` command, allows to build werf images and execute arbitrary commands in them. The difference with `werf run` is that `werf kube-run` creates a Pod in a K8s-cluster, not in a local Docker container. Using `werf kube-run` is now recommened over using `werf run`.

### Why?

* No Docker required, unlike with `werf run`.
* While running tests or similar with `werf kube-run` they are now executed in Kubernetes cluster, not on the build/local machine.  Scaling of Kubernetes nodes is usually easier.
* Much easier alternative to `kubectl run` for executing commands in images built by werf. Interactive mode is supported too.
* Alternative to `werf converge`, which would create Job for running one-time tasks in a cluster.

### Usage
While in the app git-repository:
```shell
$ werf kube-run --repo repo/repo image1 -- hostname  # non-interactively
$ werf kube-run --repo repo/repo -it image1 -- bash  # interactively
```

### How it works

By default as follows:
* Gets credentials for specified `$WERF_REPO` from `~/.docker/config.json`.
* Creates an image pull secret in a cluster with these credentials (to access private container registries).
* Creates a Pod in a cluster with the specified command, mounts the created image pull secret to the Pod.
* After Pod completed its command and stopped, created image pull secret and Pod are deleted.
